ansible_collections/arista/avd/docs/requirements.txt

```diff
@@ -3,7 +3,7 @@ mkdocs-material<10
 mkdocs-material-extensions
 pymdown-extensions
 mdx_truly_sane_lists
-mike
+mike==1.1.2
```

Comment:
lets open an issue to remove this - we should not be locking version here

Reply:
Why not? Technically they released a breaking change for us by us not pinning the version. Going from one major version to another. Ill create an issue to look at upgrading mike to 2.x standard

Reply:
Issue open #3453

Reply:
sorry - we can lock the version in the file but we should make the effort to make sure we are up to date with library otherwise we don't probably track these and we end up running very old versions of libraries. Great of the issue!